app/src/main/res/values/strings.xml

```diff
@@ -196,4 +197,20 @@
   <string name="profile_reset_pin_admin_confirm_label">Confirm 5-Digit PIN</string>
   <string name="profile_reset_pin_error_user_pin_length">Your PIN should be 3 digits long.</string>
   <string name="profile_reset_pin_error_admin_pin_length">Your PIN should be 5 digits long.</string>
+  <!-- time ago strings -->
```

Comment:
Change this to
<!-- TextViewBindingAdapters -->

Reply:
updated to   <!-- TextViewBindingAdapters -->